src/commands/CommandMusicPlay.java

```diff
@@ -28,68 +29,95 @@ public void action(){
 		}
 		else{
 			
-			if(hasParameter("l")){
-				callCommand(MUSIC_REPLAY);
+			if(hasParameter("a")){
+
+
+				String sources[][] =
+				{
+					{"V-ed's playlist on soundcloud","https://soundcloud.com/v_ed/sets/musiiic"},
+					{"No Copyrights sounds Copyright free songs playlist on youtube" , "https://www.youtube.com/watch?v=2jwj9wVx3mg&list=PLRBp0Fe2GpgnIh0AiYKh7o7HnYAej-5ph"},
+					{"No Copyrights sounds electronic playlist on youtube" , "https://www.youtube.com/watch?v=tua4SVV-GSE&list=PLRBp0Fe2GpgnZOm5rCopMAOYhZCPoUyO5"}
+					
+				};
+				
+				int rand = new Random().nextInt(sources.length);
+
+
+				MusicManager.get().loadTrack(this, sources[rand][1],
+						(player) -> connectIfNotPlaying());
+
+				sendInfoMessage( lang("Rand",sources[rand][0]),true);
```

Comment:
I would then put the `sendInfoMessage()` method call above `loadTrack()` for this change to work.

Reply:
Do you mean a message before and a message after?

Reply:
A message is already sent by the `MusicManager`'s `playlistLoaded` method https://github.com/Vhoyon/Discord-Bot/blob/6b3a3714451ddd1a86d017b3619ea511519b49c7/src/utilities/music/MusicManager.java#L118-L123
so all you'd have to do is send a message that the playlist [name of playlist] is being loaded